opentelemetry-appender-log/src/lib.rs

```diff
@@ -429,7 +429,7 @@ mod any_value {
         }
 
         fn serialize_bytes(self, v: &[u8]) -> Result<Self::Ok, Self::Error> {
-            Ok(Some(AnyValue::Bytes(v.to_owned())))
+            Ok(Some(AnyValue::Bytes(Box::new(v.to_owned()))))
```

Comment:
The side effect of boxing is that, this forces a heap allocation which maybe undesirable. I think its best to make the Boxing change separate, so that can be independently tested and reviewed.

Reply:
The boxing was done for types which are already allocated on heap. But we can discuss it independently. Will remove it

Reply:
This is removed. It does affect the bench/memory, have updated them. Would be good to bring it back.